airbyte-integrations/connector-templates/python-source/.dockerignore

```diff
@@ -1 +1,3 @@
 build
+secrets
```

Comment:
what secrets are needed in python source? also why do we have a different construction here for the .dockerignore than elsewhere? is there a reason we don't do like we do elsewhere? e.g.

```
* 
!python_source
```